benchmarks/bench.py

```diff
@@ -0,0 +1,161 @@
+from pathlib import Path
+
+import numpy as np
+import pytest
+
+from floris import (
+    FlorisModel,
+    TimeSeries,
+)
+from floris.core.turbine.operation_models import POWER_SETPOINT_DEFAULT
+from floris.heterogeneous_map import HeterogeneousMap
+
+
+N_Conditions = 100
+
+
+def test_timing_small_farm_set(benchmark):
+    """Timing test for setting up a small farm"""
+    fmodel = FlorisModel(configuration="defaults")
+    wind_directions = np.linspace(0, 360, N_Conditions)
+    wind_speeds = np.ones(N_Conditions) * 8
+    turbulence_intensities = np.ones(N_Conditions) * 0.06
+
+    benchmark(
```

Comment:
It would be good to mention how this `benchmark` function is introduced into the namespace since it isn't imported

Reply:
Added a comment above, that is a confusing practice but it's a pattern I've seen before,